src/container.c

```diff
@@ -21,6 +21,7 @@
 #include <string.h>
 #include "dispatch_private.h"
 #include "connection_manager_private.h"
+#include "server_private.h"
```

Comment:
It would be better to not add this private linkage between container and server.  Rather, put the signature for qd_connection_set_user into the public server.h file.